src/styles/main_page/_GongZhongHao.scss

```diff
@@ -0,0 +1,37 @@
+//@import "../../main_page/GongZhongHao.js";
+
+.MainGrid { 
+    display: grid;
+    //grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
+    //grid-template-columns: 400px 400px 400px;
+    grid-template-columns: repeat(auto-fill, 370px);
+    grid-template-rows: repeat(auto-fill, 290px);
+    grid-gap: 10px;
```

Comment:
We want this section to be horizontally scrollable. Grid might not be the optimal solution. See if the following links help.
https://css-tricks.com/pure-css-horizontal-scrolling/
https://codeburst.io/how-to-create-horizontal-scrolling-containers-d8069651e9c6
https://uxdesign.cc/creating-horizontal-scrolling-containers-the-right-way-css-grid-c256f64fc585